test/testsuite.h

```diff
@@ -29,6 +29,8 @@
 
 class options;
 
+std::string writestr(const std::string& str, bool gccStyle = false);
```

Comment:
I'd prefer that `writestr` is not a global scope function. If you have to make it non-local then I think you should be able to convert it into a private method.